### ISSUE TITLE
0.0.5 unusable, can't click anywhere

### ISSUE THREAD

**Issue by deboer-tim** [containers/podman-desktop#265]
### Bug description

I just updated to 0.0.5 and can't get to the Images, Containers, or Preferences page anymore. Clicking on the tabs does nothing.

I can see the new welcome page and it appears to work - external links function, clicking to start or using toolbar menu to stop podman seems to work. It doesn't refresh though, e.g. if I start podman I need to wait and restart desktop to see that it has started.

### Operating system

macOS 12.4 M1

### Version

next (development version)

### Steps to reproduce

Just installed 0.0.5 universal image and launched. Using an M1.

**Comment by deboer-tim** [comment]
FWIW I tried reinstalling and switched to the arm package, same behaviour.

**Comment by slemeur** [comment]
I have the same issue. But I think the issue happens only after it gets status back from CRC. 


https://user-images.githubusercontent.com/1636769/177338833-42444c1b-2f28-45ae-a423-7521f92c6abf.mp4

**Comment by gbraad** [comment]
an issue with a call (thread-model) blocking the UI?

**Comment by slemeur** [comment]
When CRC is running, I have the same issue.

**Comment by slemeur** [comment]
If CRC extension is disabled (from the extension catalog in the preferences), I don't have the issue.

**Comment by deboer-tim** [comment]
Also confirmed - but for me it re-enables every time I start (I will open another issue). :(

**Comment by gbraad** [comment]
I remember @evidolob mention that the extensions are limited in 'allowed' behaviour. And for CRC this status is requested with a rest call to the backend, instead of a cmd run. So perhaps this causes a block
